ilmercatinodellibro/ilmercatinodellibro
# feat(client+server): add settlement user filters

fixes #190

## What it does
This feature adds two new filters to the Users Management page:
- Users with settleable books
- Already settled users
The filters are mutually inclusive

## How to test it
Open the Users Management page, settle some users, check the filters and check that the users that are shown are correct
